app/src/main/res/navigation/profile.xml

```diff
@@ -74,5 +78,11 @@
         android:name="org.brightmindenrichment.street_care.ProfileMyEvents"
         android:label="@string/your_events"
         tools:layout="@layout/fragment_profile_my_events" />
+    <fragment
+        android:id="@+id/nav_changepassword"
+        android:name="org.brightmindenrichment.street_care.ui.user.ChangePasswordFragment"
+        android:label="@string/profile"
```

Comment:
This should display `Change Password` as the fragment title and then remove the additional header placed in the xml view that duplicates this title. See the figma design below

<img width="331" alt="image" src="https://github.com/user-attachments/assets/ab390c34-a47c-4216-a254-40ca85a3c905">